ceph/ceph/upgrade.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+NAMESPACE=${NAMESPACE:-"ceph"}
+TIMEOUT=${TIMEOUT:-10}
+UPGRADE_ORDER="mgr mon osd mds rgw rbd-provisioner"
+KUBECTL="kubectl"
+
+function wait_for_pod_ready {
+    pod=$1
+    i=${TIMEOUT}
+    echo "Wait for Pod "$pod" to be ready"
```

Comment:
Using double quotes inside double quotes is not necessary, the shell will interpret.